knowledge-base/kb-chartview-display-multiple-donut-series.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to Display Multiple Doughnut Series
+description: How to show several pie charts in RadChartView
+type: how-to
+page_title: Plot Multiple Donuts
+slug: kb-chartview-display-multiple-donut-series
+position: 0
+tags: pie, douhgnut, visualization, data, radpiechart
+ticketid: 831081
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.1.220</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadChartView for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to display multiple DoughnutSeries in a single pie chart, without overlapping.
+
+## Solution
+
+Adjust the series positions using their [RadiusFactor and InnerRadiusFactor]({%slug radchartview-series-doughnutseries%}#setting-the-doughnut-radius) properties.
```

Comment:
We should add an apostrophe after series- the series__'__ positions, or rephrase a bit. For example: adjust the positions of the series.